lib/get-file-paths.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 /**
  * Returns an object of file path data
- * 
+ *
  * @param {object}  file  a vinyl file object
  */
 exports.getFilePaths = function (file) {
```

Comment:
@ericduran @scottnath I don't think this module isn't all that complex or magical, although it can be optimized. You're repeating file.path.replace(process.cwd() + '/', ''), so set is as a var like so:
`var rel = file.path.replace(process.cwd() + '/', '');`
Then, update the two lines in the paths object:
`relative: rel,`
`directory: path.basename(path.dirname(rel))`